components/brave_new_tab_ui/components/default/braveToday/cardsGroup.tsx

```diff
@@ -121,7 +121,10 @@ export default function CardsGroups (props: Props) {
       >
         {cardInstance}
         {shouldTriggerViewCountUpdate &&
-          <div {...{ [attributeNameCardCount]: index }} ref={props.onPeriodicCardViews} />
+          <div
+            {...{ [attributeNameCardCount]: index + props.itemStartingDisplayIndex }}
+            ref={props.onPeriodicCardViews}
+          />
```

Comment:
It's a bug fix. Not sure if this was due to some recent change, but the front-end was only reporting the total visible cards in the current page. This fix ensures that we get a total count for all visible pages.

Reply:
Ok, thanks. It would be better to split this into a separate commit so the rationale can be documented, even if it's not big enough to open a separate issue for it.